### PR TITLE
Adding June Swagger Update - Azure Container Instance

### PR DESCRIPTION
**Pull request by samkreter**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by lmazuel** [comment]
Hi @samkreter !
Please update the Readme to add a new tag and change "Basic Information" to point to this new tag. Then the bot generation will be meaningfull. Thanks!

**Review by marstr — CHANGES_REQUESTED** [review]
Mind updating the Golang config in the README as well? https://github.com/Azure/azure-rest-api-specs/pull/3170/files#diff-5c3d140eca9525738ed2ab683fb322afR155

**Review by marstr — APPROVED** [review]
Now we're just waiting for @ravbhatnagar

File: specification/containerinstance/resource-manager/Microsoft.ContainerInstance/stable/2018-06-01/containerInstance.json [hunk 1]
```diff
@@ -0,0 +1,1355 @@
...
+            "schema": {
+              "$ref": "#/definitions/UsageListResult"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerInstance/containerGroups/{containerGroupName}/containers/{containerName}/logs": {
+      "get": {
+        "operationId": "ContainerLogs_List",
```
Comment (marstr) [inline specification/containerinstance/resource-manager/Microsoft.ContainerInstance/stable/2018-06-01/containerInstance.json]: There are some semantics that we introduce to OperationIDs in AutoRest. Chiefly, we use the prefix here to denote an "operation group". For some languages, each operation group gets is own Client type.

For that reason, I'd recommend having fewer Operation Groups:

`ContainerGroupUsage_List` -> `ContainerGroups_ListUsage`
`ContainerLogs_List` -> `Container_ListLogs`
`StartContainer_LaunchExec` -> `Container_StartExecLaunch`

**Review by marstr — CHANGES_REQUESTED** [review]

File: specification/containerinstance/resource-manager/Microsoft.ContainerInstance/stable/2018-06-01/containerInstance.json (hunk 1 above)
Comment (marstr) [reply specification/containerinstance/resource-manager/Microsoft.ContainerInstance/stable/2018-06-01/containerInstance.json]: I took another look and found another thing we may want to tweak, not necessary to fix it though! Just acknowledge if you don't want to fix it. :)

Comment (samkreter) [reply specification/containerinstance/resource-manager/Microsoft.ContainerInstance/stable/2018-06-01/containerInstance.json]: @marstr I would rather save those for the next PR so we can get this June release out. I can update those changes quickly for another PR.

Comment (marstr) [reply specification/containerinstance/resource-manager/Microsoft.ContainerInstance/stable/2018-06-01/containerInstance.json]: It'd be a breaking change in the future, so we should do it now or never. Your choice though. :)

**Review by marstr — APPROVED** [review]
~~When comparing the two API Versions against one another, this is actually a relatively small change. Nothing troublesome jumped out to me.~~ 

edit: wrong window, though the approval counts.

**Pull request merged**